src/extension.ts

```diff
@@ -289,16 +290,29 @@ export async function launch(context: vscode.ExtensionContext, launchUrl?: strin
             telemetryReporter.sendTelemetryEvent('command/launch/browser', browserProps);
 
         browserInstance = await launchBrowser(browserPath, port, url, userDataDir);
-        browserInstance.addListener('targetcreated', () => {
+        if (url !== SETTINGS_DEFAULT_URL) {
+            reportIsLocalhost(url);
+        }
+        browserInstance.on('targetcreated', () => {
             cdpTargetsProvider.refresh();
         });
-        browserInstance.addListener('targetdestroyed', () => {
+        browserInstance.on('targetdestroyed', () => {
```

Comment:
What are the functional differences of swapping out the `addListener` with `on`?

Reply:
Ah, just saw the deprecated note on `addListener`